templates/keepalived.conf.j2

```diff
@@ -168,39 +168,133 @@ virtual_server {{ vserver.ip }} {{ vserver.port }} {
   {% for rserver in vserver.real_servers %}
   real_server {{ rserver.ip }} {{ rserver.port }} {
     {% if rserver.tcp_checks is defined %}
-    {% for tcp_check in rserver.tcp_checks %}
-    TCP_CHECK {
-      connect_port {{ tcp_check.connect_port }}
-      connect_timeout {{ tcp_check.connect_timeout | default('5') }}
-    }
-    {% endfor %}
+      {% for tcp_check in rserver.tcp_checks %}
```

Comment:
I am fine with this, however, there seems to be an indentation difference. People using tcp_checks will see a diff, which will basically restart their keepalived, even if there is no functional change for them...

While the template looks cleaner this way, I think the file will contain extra spaces. Don't you think we should keep the same indentation as right now in the generated file?

Reply:
yes there will be an extra space in the config file. i've just added the extra spaces during debugging of a jinja template error. i'll change that back to the default spacing.

Reply:
Awesome, thanks!